Mathlib/Topology/Compactness/Lindelof.lean

```diff
@@ -0,0 +1,50 @@
+/-
+Copyright (c) 2023 Josha Dekker. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Josha Dekker
+-/
+import Mathlib.Topology.Bases
+import Mathlib.Order.Filter.CountableInter
+/-!
+# Compact sets and compact spaces
+
+## Main definitions
+
+We define the following properties for sets in a topological space:
+
+* `IsLindelof`: a set such that each open cover has a countable subcover. This is defined in mathlib
+  using filters.
+* `LindelofSpace`: typeclass stating that the whole space is a Lindëlof set.
+* `NonLindelofSpace`: a space that is not a Lindëlof space.
+
+## Main results
+
+* ToBeAdded
+-/
+open Set Filter Topology TopologicalSpace Classical
```

Comment:
Do you really need to open `Classical`?
```suggestion

open Set Filter Topology TopologicalSpace

Reply:
Not yet, but I've skimmed through the first dozen theorems in Compact, and they seem to carry over relatively easily to Lindelöf, so I thought I'd just leave Classical in there already; but you're right, it is not necessary as is.